### DISCUSSION TITLE
export to SBML

### DISCUSSION THREAD

**Discussion by martinjrobins** [MichaelClerx/myokit#1089] [Q&A] [answered]
Hi, I was wondering if it were possible to export a myokit model to SBML. I can see an importer, but not an exporter. Perhaps it can be done indirectly? e.g. mmt -> cellml -> sbml?

**Comment by MichaelClerx** [comment, marked as answer]
Hi Martin,
There's no SBML export in Myokit at the moment.
I think there are CellML to SBML converters, so that route is an option if you're in a hurry!
Otherwise, it wouldn't be too hard to add SBML export. There is already an intermediate description (objects representing an SBML model). So you'd need to write (A) something that turns a myokit.Model into a myokit.formats.SBML.model, and (B) something to write SBML models as XML. Neither should be an issue, & I'm happy to help if anyone wants to take it on

**Reply by martinjrobins** [reply]
Thanks @MichaelClerx, I don't think I've got time to write this atm so might try the cellml to sbml route for now, but will return to it if  I can

**Reply by MichaelClerx** [reply]
Cool! Would you mind making an issue for it already? Just so I have a record of user interest 😄

**Reply by martinjrobins** [reply]
done :) https://github.com/myokit/myokit/issues/1091
